playground/withdockerfile/WithDockerfile.AppHost/qots/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Stage 1: Build the Go program
+ARG GO_VERSION=1.16
+FROM golang:${GO_VERSION} AS builder
+WORKDIR /app
+COPY . .
+RUN go build qots.go
+
+# Stage 2: Run the Go program
+ARG GO_VERSION
+FROM golang:${GO_VERSION}
+WORKDIR /app
+RUN --mount=type=secret,id=SECRET_ASENV cp /run/secrets/SECRET_ASENV /app/SECRET_ASENV
+RUN --mount=type=secret,id=SECRET_ASFILE cp /run/secrets/SECRET_ASFILE /app/SECRET_ASFILE
```

Comment:
Do these get read anywhere

Reply:
Not in the playground. I just use docker desktop to inspect the values to understand what is going on.